sentry_sdk/profiler.py

```diff
@@ -374,34 +380,37 @@ def teardown(self):
 
     def start_profiling(self):
         # type: () -> bool
-        if super(_ThreadScheduler, self).start_profiling():
+        if super(ThreadScheduler, self).start_profiling():
             # make sure to clear the event as we reuse the same event
             # over the lifetime of the scheduler
-            self.event.clear()
+            event = threading.Event()
+            self.threads.put_nowait(event)
```

Comment:
just nitpicking but could we possible have a better name than `threads`?
Maybe signify purpose, like `termination_events/stop_events` or something since we use them to break the profiler loops?